### PR TITLE
drop from root in entrypoint script

### LINKED ISSUE #68: Run thelounge as unprivileged in the docker container
Ref IRC talk. PR incoming.

### PR DESCRIPTION
**Pull request by williamboman**
Closes #68.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -10,24 +10,13 @@ EOF
...
-    # optional flags defined via environment variables
-    exec "$@" \
-        $CONF_OPT_HOST \
-        $CONF_OPT_PORT \
-        $CONF_OPT_BIND \
-        ;
+if [ "$(id -u)" = '0' ]; then
+    # use the node user by default
+    if [ "$*" = "thelounge start" ]; then
+        find "${THELOUNGE_HOME}" \! -user node -exec chown node '{}' +
```
Comment (xPaw) [inline docker-entrypoint.sh]: Why not `chown --recursive` ?

Comment (williamboman) [reply docker-entrypoint.sh]: https://github.com/docker-library/redis/pull/166

**Comment by capi** [comment]
Thanks!

**Pull request merged**